### PR TITLE
Remove comments from project.json

### PR DESCRIPTION
**Pull request by troydai**
Comment support will be removed from `project.json`

https://github.com/aspnet/dnx/pull/1779

### CODE REVIEW

File: samples/SocialSample/project.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
         "Kestrel": "1.0.0-*"
     },
     "commands": {
-        /* Note all servers must use the same address and port because these are pre-registered with the various providers. */
```
Comment (Tratcher) [inline samples/SocialSample/project.json]: Please move this comment into the Startup file.

**Comment by troydai** [comment]
@Tratcher better?

**Comment by Tratcher** [comment]
Yes, thanks.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**